test/integration.js

```diff
@@ -7,6 +7,24 @@ const c = require('../constants')
 
 let dc = null
 
+function configureDefaultDC (dcInstance) {
```

Comment:
Can call this parameter `dc`.

Reply:
Can confuse with global dc but yes.

Reply:
No, it can't confuse :) It's a single parameter in a one line function.

Reply:
`dc`, it's not _super_ important